.ci/install_crio.sh

```diff
@@ -64,14 +64,9 @@ then
 fi
 
 echo "Get CRI Tools"
-critools_repo="${kubernetes_sigs_org}/cri-tools"
-go get "$critools_repo" || true
-pushd "${GOPATH}/src/${critools_repo}"
-critools_version=$(grep "ENV CRICTL_COMMIT" "${GOPATH}/src/${crio_repo}/Dockerfile" | cut -d " " -f3)
-git checkout "${critools_version}"
-go install ./cmd/crictl
-sudo install "${GOPATH}/bin/crictl" /usr/bin
-popd
+crictl_version=v1.14.0
+wget -qO- https://github.com/kubernetes-sigs/cri-tools/releases/download/$crictl_version/crictl-$crictl_version-linux-amd64.tar.gz \
```

Comment:
@chavafg -do we only test on x86 - or do we need to make this arch independent/tolerant ?

Reply:
oh, you are right... We currently test only x86, but I know that @Pennyzct is adding support for arm, so I think it would be good to add support for different architectures.
@saschagrunert do you think you can make the change?  
I think you can use `.ci/kata-arch.sh -g` to get the correct arch name.

Reply:
Alright, I will do it like this.